Formula/apib.rb

```diff
@@ -1,9 +1,9 @@
 class Apib < Formula
   desc "HTTP performance-testing tool"
   homepage "https://github.com/apigee/apib"
-  url "https://github.com/apigee/apib/archive/APIB_1_0.tar.gz"
-  sha256 "1592e55c01f2f9bc8085b39f09c49cd7b786b6fb6d02441ca665eef262e7b87e"
-  revision 3
+  url "https://github.com/apigee/apib/archive/APIB_1_2.tar.gz"
+  sha256 "f2ed580a84ceb456ab3ca6e81c14e61b13c067dc447c65dd8f63dc0e8db15244"
+  revision 1
```

Comment:
This shouldn't be here for new versions

Reply:
Sorry, removed revision in the latest commit.